meilisearch/client.py

```diff
@@ -672,3 +673,21 @@ def _valid_uuid(uuid: str) -> bool:
         )
         match = uuid4hex.match(uuid)
         return bool(match)
+
+    def create_snapshot(self) -> TaskInfo:
```

Comment:
Lets move this method up above the private methods. You can put it right after `create_dump`.

Reply:
Moved it right after create_dump